dist/actions/entrypoint.sh

```diff
@@ -82,12 +82,14 @@ if [ -e package.json ] && [ ! -d node_modules ]; then
     npm install
 fi
 
-# Now just pass along all arguments to the Pulumi CLI.
-OUTPUT=$(sh -c "pulumi --non-interactive $*" 2>&1)
-EXIT_CODE=$?
-
-echo "#### :tropical_drink: \`pulumi ${@:2}\`"
-echo "$OUTPUT"
+# Now just pass along all arguments to the Pulumi CLI, sending the output to a file for
+# later use. Note that we exit immediately on failure (under set -e), so we `tee` stdout, but
+# allow errors to be surfaced in the Actions log.
+PULUMI_COMMAND="pulumi $*"
+OUTPUT_FILE="/tmp/out.txt"
```

Comment:
Can we use something slightly more specific, or can this be whatever we want?

Perhaps `/tmp/pulumi-output.txt`? Or, perhaps even better, `OUTPUT_FILE=$(mktemp -t Pulumi-output)`? (The latter creating a random file each time, so each invocation is different and you can run the container multiple times on the same system without worries.

Reply:
Ah, I always forget about `mktemp`. I'll use that and rename to `pulumi-output`, too.